src/styles/components/_calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  width: ($day-size * $week-long) + ($cell-gap * ($week-long - 1));
+  padding: 10px;
+  gap: $cell-gap;
+  flex-wrap: wrap;
+  list-style: none;
+
+  @each $day, $day-num in $week-days {
```

Comment:
Would be more correct to write `$index` instead of `$day-num` because it starts from 0

Reply:
done!